pyproject.toml

```diff
@@ -0,0 +1,9 @@
+# NOTE: this pyproject.toml is only defined in the repository because it's the only official
+# default configuration file for black [1] that the tool walks the directory hierarchy upwards
+# automatically (until it finds .git|.hg) without us having to specify a config file manually
+# (twice) and since there are at least 5 Python "sub-packages" in this repository, the black
+# settings should be uniform given it's tested by tox.
+#
+# [1] https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file
+[tool.black]
+line_length = 120
```

Comment:
To be honest, I personally would like to keep the line length in a reasonable number as much as possible so that I don't have to read the code from screen left to the right. Wrapping lines properly increases the readability. Meanwhile, by setting a value to `line_length`, black seems not allow the wrap lines manually by not exceeding the value. That loose flexibility, I think. What do you think to keep the flexibility for each sub-module to configure such preference for themselves?